rexmex/metrics/classification.py

```diff
@@ -363,6 +485,11 @@ def threat_score(y_true: np.array, y_score: np.array) -> float:
     return ts
 
 
+@classifications.annotate(  # TODO
```

Comment:
help wanted - I'm not really sure what the bounds for this are nor whether bigger is better